editor/modes/visual-editor/invalid-block-warning.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { Dashicon } from 'components';
+import { __ } from 'i18n';
+
+const warning = (
+	<div className="editor-visual-editor__invalid-block-warning">
+		<Dashicon icon="warning" size={ 40 } />
+		<p>{ __(
+			'Uh oh! This block is corrupt and has been locked to protect ' +
```

Comment:
"This block has been modified externally and has been locked to protect against content loss." cc @jasmussen